src/components/SingleEvent/EventParticipants.js

```diff
@@ -21,12 +21,17 @@ const Spots = styled('span')`
 
 class EventParticipants extends Component {
   state = {
-    search: ''
+    search: '',
+    selectedFilter: null
   }
 
-  handleSearch = search => {
+  handleFilterChange = selectedFilter => {
+    this.setState({ selectedFilter })
+  }
+
+  handleSearch = event => {
     this.setState({
-      search: search || ''
+      search: event.target.value
```

Comment:
`handleSearch` is no longer called with the event, but the string itself. This needs to be as it was before, otherwise QRcode scanner callback won't work.

Reply:
Yep, just confirmed on your PR that search keyword filtering is broken. We need end-to-end tests!

Reply:
I'll sort this out